etc/grafana/dashboards/reth-mempool.json

```diff
@@ -2679,6 +2679,176 @@
       "title": "Fetch Hashes Pending Fetch Duration",
       "type": "timeseries"
     },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "${DS_PROMETHEUS}"
```

Comment:
nice, see now that you exported it for public sharing, that this is a var not a literal

Reply:
yes, sorry for that, wait i'll fix plugin versions

Reply:
also screenshot of the panel would be nice :)

Reply:
https://snapshots.raintank.io/dashboard/snapshot/fgtPEIpc4dKvEZqiajx6sW5jkEWxaJ5e

panel is empty because metrics is not exposed, i've tried to get it with `curl` still no luck

Reply:
![Uploading zeroMetrics.png…]()

Reply:
maybe need to restart node to run off latest main

Reply:
Running latest version
```
# reth --version
reth Version: 1.0.8
Commit SHA: 6c16df236567047ea62a4750197e1847771d20d7
Build Timestamp: 2024-10-03T16:17:27.298425557Z
Build Features: asm_keccak,jemalloc
Build Profile: maxperf
```